src/metabase/models/metric.clj

```diff
@@ -30,30 +30,30 @@
 (def Metric
```

Comment:
I was thinking we should rename this and rename the namespaces as well e.g. `LegacyMetric` and `metabase.models.legacy-metric`.

Ideally we'd even rename the API endpoints, it wouldn't be too hard to update the FE code that calls them since they're all in one place IIRC. But maybe that's less important since Metrics v1 are going away soon anyway